server/main.py

```diff
@@ -81,21 +98,26 @@ def disconnect(self, client: Client, room: str) -> None:
             client: The Client to which the connection belongs.
             room: The room from which the client will be disconnected.
         """
-        self._rooms[room].remove(client)
+        self._rooms[room]["clients"].remove(client)
 
-        if len(self._rooms[room]) == 0:
+        if len(self._rooms[room]["clients"]) == 0:
             del self._rooms[room]
 
-    async def broadcast(self, data: dict, room: str) -> None:
+    async def broadcast(self, data: dict, room: str, everyone=False) -> None:
```

Comment:
Type hinting for the `everyone` parameter can be added, just to be consistent with the rest of the file.

Reply:
Yup, completely forgot 😆